client/components/checklist/index.js

```diff
@@ -0,0 +1,68 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import classNames from 'classnames';
+import React, { Children, PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import { times } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import ChecklistHeader from 'components/checklist/header';
+import TaskPlaceholder from 'components/checklist/task-placeholder';
+
+export default class Checklist extends PureComponent {
+	static propTypes = {
+		isPlaceholder: PropTypes.bool,
+	};
+
+	state = { hideCompleted: false };
+
+	toggleCompleted = () =>
+		this.setState( ( { hideCompleted } ) => ( { hideCompleted: ! hideCompleted } ) );
+
+	renderPlaceholder() {
+		return (
+			<div className={ classNames( 'checklist', 'is-expanded', 'is-placeholder' ) }>
```

Comment:
No need for `classNames` here to generate static class. However, the visibility toggle doesn't work on the placeholder, but it could. Let's leave this as it will be useful then, and handle that in a follow-up PR.